dts/arm/nordic/nrf51822.dtsi

```diff
@@ -46,6 +46,25 @@
 			label = "UART_0";
 		};
 
+		gpiote: gpiote@40006000 {
+			compatible = "nordic,nrf5-gpiote";
+			reg = <0x40006000 0x1000>;
+			interrupts = <6 1>;
+			interrupt-names = "gpiote";
+			status = "disabled";
+			label = "GPIOTE_0";
+		};
+
+		gpio0: gpio@50000000 {
+			compatible = "nordic,nrf5-gpio";
+			gpio-controller;
+			reg = <0x50000000 0x200
```

Comment:
Something looks wrong, here: why do we have two lines for region range? This should be for nRF52840 that has P0 and P1.

Reply:
I created it based on dts for nRF52832, I will change it to one region